tests/Unit/Core/Localization/CLDR/CurrencyTest.php

```diff
@@ -52,7 +52,7 @@ protected function setUp(): void
     {
         $currencyData = new CurrencyData();
         $currencyData->setIsoCode('PCE');
-        $currencyData->setNumericIsoCode(333);
+        $currencyData->setNumericIsoCode('333');
```

Comment:
this modification is breaking the unit test, you should probably set the numeric iso code as a string at line 84 too

Reply:
Just fixed.